packages/@lightningjs/ui-components/src/components/InlineContent/InlineContent.js

```diff
@@ -360,6 +359,26 @@ export default class InlineContent extends Base {
         text: text.text || text
       }
     };
+    if (textOverrideStyles?.textDecoration === 'line-through') {
+      const textWidth = measureTextWidth({
+        ...this.style.textStyle,
+        ...textOverrideStyles,
+        text: text.text || text
+      });
+      const strikethroughLine = {
+        rect: true,
+        w: textWidth - 2,
+        h: 2,
+        color: textOverrideStyles?.textColor || 0xffffffff,
+        y: textComponent.h / 2
+      };
+      return {
+        type: lng.Component,
+        w: textWidth + 5,
```

Comment:
Adding `h: textComponent.h` here will resolve the overlapping issue when contentWrap is true.